src/core/reaction_ensemble.hpp

```diff
@@ -109,7 +110,17 @@ struct DegreeOfAssociationCollectiveVariable : public CollectiveVariable {
 class ReactionAlgorithm {
 
 public:
-  ReactionAlgorithm() = default;
+  ReactionAlgorithm() = delete;
+  ReactionAlgorithm(int seed) {
+    std::seed_seq seeder{
+        seed}; // come up with "sane" initialization to avoid too many zeros in
+               // the internal state of the Mersenne twister
+    m_generator.seed(seed);
+    m_generator.discard(1e6); // discard the first 1e6 random numbers to warm
+                              // up the Mersenne-Twister PRNG
+    std::normal_distribution<double> m_normal_distribution(0, 1);
```

Comment:
This declares local variables inside of the constructor, are you sure that is what you want?